src/main/java/spireMapOverhaul/zones/voidseed/campfire/CampfireCorruptEffect.java

```diff
@@ -0,0 +1,106 @@
+package spireMapOverhaul.zones.voidseed.campfire;
+
+import basemod.helpers.CardModifierManager;
+import com.badlogic.gdx.Gdx;
+import com.badlogic.gdx.graphics.Color;
+import com.badlogic.gdx.graphics.g2d.SpriteBatch;
+import com.badlogic.gdx.math.Interpolation;
+import com.megacrit.cardcrawl.cards.AbstractCard;
+import com.megacrit.cardcrawl.cards.CardGroup;
+import com.megacrit.cardcrawl.core.CardCrawlGame;
+import com.megacrit.cardcrawl.core.Settings;
+import com.megacrit.cardcrawl.dungeons.AbstractDungeon;
+import com.megacrit.cardcrawl.helpers.ImageMaster;
+import com.megacrit.cardcrawl.localization.UIStrings;
+import com.megacrit.cardcrawl.rooms.AbstractRoom;
+import com.megacrit.cardcrawl.rooms.CampfireUI;
+import com.megacrit.cardcrawl.rooms.RestRoom;
+import com.megacrit.cardcrawl.vfx.AbstractGameEffect;
+import com.megacrit.cardcrawl.vfx.cardManip.ShowCardBrieflyEffect;
+import spireMapOverhaul.zones.voidseed.cardmods.CorruptedModifier;
+
+import java.util.Iterator;
+
+import static spireMapOverhaul.SpireAnniversary6Mod.makeID;
+
+public class CampfireCorruptEffect extends AbstractGameEffect {
```

Comment:
Please stop copying decompiled code from intellih's decompiler, it's killing me. At least copy it from jdgui or somethign so it isn't as obvious.